docs/multitool-usage.md

```diff
@@ -56,7 +56,7 @@ Sarif.Multitool export-validation-config validation.xml
 Sarif.Multitool export-validation-rules ValidationRules.md
 
 : Merge multiple SARIF files into one
-Sarif.Multitool merge C:\Input\*.sarif --recurse --output-directory=C:\Output\ --output-file=MergeResult.sarif
+Sarif.Multitool merge C:\Input\*.sarif --recurse true --output-directory=C:\Output\ --output-file=MergeResult.sarif
```

Comment:
>true [](http://example.com/codeflow?start=49&length=4)

This shouldn't be literally required, you *can* provide an explicit true but you can also exclude it. Do you not observe that this is correct in your testing?